docs/project_defn.rst

```diff
@@ -122,8 +122,10 @@ following properties:
           Since there can be multiple costs, this permits both fixed and variable costs for upgrades
           that depend on the properties of the baseline building.
           The multiplier needs to be from
-          `this enumeration list inthe resstock or comstock repo <https://github.com/NREL/resstock/blob/master/measures/ApplyUpgrade/measure.rb#L71-L87>`_
-          or from the list in your branch of that repo.
+          this enumeration list in the
+          `resstock <https://github.com/NREL/resstock/blob/main/measures/ApplyUpgrade/resources/constants.rb#L12-L38>`_ or
```

Comment:
Alternatively, it may be appropriate to point to resstock's RTD that displays cost multiplier types: https://resstock.readthedocs.io/en/latest/workflow_outputs/default_outputs.html#upgrade-costs

Reply:
Works for me. Can you make that change?

Reply:
Wait a minute. But those aren't the enumerations that one would put in the input file. That's the context here. If someone tries to put in `upgrade_costs.door_area_ft_2` instead of `Door Area (ft^2)` it won't work.

Reply:
Good point. The RTD should probably include the exact enums. But out of scope here. Let's just stick with pointing to `constants.rb`.